src/Sylius/Bundle/ReviewBundle/spec/EventListener/ReviewChangeListenerSpec.php

```diff
@@ -33,11 +34,11 @@ function let(ReviewableRatingUpdaterInterface $averageRatingUpdater): void
 
     function it_recalculates_subject_rating_on_accepted_review_deletion(
         $averageRatingUpdater,
-        GenericEvent $event,
+        LifecycleEventArgs $event,
         ReviewInterface $review,
         ReviewableInterface $reviewSubject
     ): void {
-        $event->getSubject()->willReturn($review);
+        $event->getObject()->willReturn($review);
         $review->getStatus()->willReturn(ReviewInterface::STATUS_ACCEPTED);
```

Comment:
This one is not needed now as well since review status does not matter in this listener :)